incuna-sass/helpers/_link.sass

```diff
@@ -1,16 +1,16 @@
 // Mixin to quickly add / remove default styles to links
 @mixin link($type: default)
     cursor: pointer
-    @if $type = default
```

Comment:
For every use of this mixin, the styles will be the same. Before, the styles that would be rendered for all uses were the `blank` styles; now they are the `default` styles.